config/locales/candidate_interface/authentication.yml

```diff
@@ -18,7 +18,7 @@ en:
       email:
         subject: Check the email address you’re using to apply for teacher training
     expired_token:
-      heading: The link you clicked has expired
+      heading: The link you selected has expired
```

Comment:
I'm not convinced that 'selected' is quite right here. Why did we make the change from clicked in this instance?

Reply:
It's generally a word we try and avoid because not everyone 'clicks' a link e.g. if they can't use a mouse and use a keyboard to navigate.

Reply:
That's fair enough and makes sense. Let's change then!

Reply:
`select` does strike me as slightly odd. how about `follow`?

Reply:
The content hive mind has suggested: 'The link you used to sign in has expired' - which ties it in with the email they would have just seen which mentions Sign in. So it flows a bit better. When in doubt, go to the hive mind.

Reply:
Nice one - you okay to update this and the tests?

Reply:
Yup!